checker/jtreg/nullness/type-argument/TypeArgument.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * @test
+ * @summary Test case for type argument in method invocation.
+ *
+ * @compile/fail/ref=TypeArgument.out -XDrawDiagnostics -processor org.checkerframework.checker.nullness.NullnessChecker TypeArgument.java
+ */
+public class TypeArgument<T> {
```

Comment:
`TypeArgument` is not a very descriptive name for this test.

Reply:
Renamed to `NonRawTypeArgumentTest`